vulture/core.py

```diff
@@ -217,7 +218,7 @@ def get_list(typ):
 
         self.filename = Path()
         self.code = []
-        self.found_dead_code_or_error = False
+        self.found_dead_code_or_error = ExitCode.NoDeadCode
```

Comment:
```suggestion
        self.exit_code = ExitCode.NoDeadCode
```
Let's rename the variable.

Reply:
I made the suggested changes. thanks!